pdns/ws-auth.cc

```diff
@@ -1153,6 +1154,7 @@ static void apiZoneCryptokeysPOST(DNSName zonename, HttpRequest *req, HttpRespon
     privatekey_fieldname = "content";
   }
   bool active = boolFromJson(document, "active", false);
+  bool published = boolFromJson(document, "published", false);
```

Comment:
shouldn't this default to `true`, to avoid surprises for API users who don't know about `published`?

Reply:
Fixed.